parserc.go

```diff
@@ -308,6 +308,7 @@ func (parser *yamlParser) parseDocumentStart(event *yamlEvent, implicit bool) bo
 			typ:        yaml_DOCUMENT_START_EVENT,
 			start_mark: token.start_mark,
 			end_mark:   token.end_mark,
+			implicit:   true,
```

Comment:
I'm unsure what this change is about.

Could you elaborate a bit here?

Thanks

Reply:
The `implicit` flag is used to mark document start and end events, to distinguish between explicit and implicit versions. In all other three cases (explicit document start, implicit document end, implicit document start) it is set, but for an implicit document start it is not set (and thus defaults to `false`). I don't think this is used anywhere (except when emitting the stream of events again), but this is IMO a clear bug that should be fixed.

Reply:
In case anyone wonders where this part went: #138.